docs/en/siem/rules-ui-create.asciidoc

```diff
@@ -0,0 +1,141 @@
+. Use the filter and query fields to create the criteria used for detecting 
+signals. For example, a rule that detects when a Microsoft Office product 
+starts a process in a user folder uses the following query and filters:
+
+* _Custom query_ `process.parent.name: EXCEL.EXE or process.parent.name: MSPUB.EXE or process.parent.name: OUTLOOK.EXE or process.parent.name: POWERPNT.EXE or process.parent.name: VISIO.EXE or process.parent.name: WINWORD.EXE`
+* _Add filter_:
+** *Field*: `event.action`
+** *Operator*: `is`
+** *Value*: `Process Create (rule: ProcessCreate)`
+. Click *Continue*.
++
+The *About rule* pane is displayed.
+[role="screenshot"]
+image::about-rule-pane.png[]
+. Fill in the following fields:
+.. _Name_: The rule's name.
+.. _Description_: A description of what the rule does.
+.. _Severity_: Select the severity levels of signals created by the rule:
+* `Low`: Signals that are of interest but generally not considered to be 
+security incidents. Sometimes, a combination of low severity events can 
+indicate suspicious activity.
+* `Medium`: Signals that require investigation.
+* `High`: Signals that require an immediate investigation.
+* `Critical`: Signals that indicate a security incident has occurred.
+.. _Risk score_: A numerical value between 0 and 100 that correlates with the _Severity_ level. General guidelines are:
+* `21` represents low severity.
+* `47` represents medium severity.
+* `73` represents high severity.
+* `99` represents critical severity.
+.. For additional options, click *Advanced settings* and fill in any of 
+these fields: 
+... _Timeline template_ (optional): Select a timeline template used when you 
+send a signal created by the rule to the timeline.
++
+TIP: Before you create rules, create and save relevant
+<<timelines-ui,timelines>> so they can be selected here. 
+
+... _Reference URLs_ (optional): References to information that is relevant to 
+the rule. For example, links to relevant background information.
+... _False positives_ (optional): List of common scenarios that may produce 
+false-positive signals.
+... _MITRE ATT&CK^TM^_ (optional): Relevant MITRE framework tactics and techniques.
+... _Tags_ (optional): Words and phrases used to categorize, filter, and search 
+the rule.
+. Click *Continue*.
++
+The *Schedule rule* pane is displayed.
+[role="screenshot"]
+image::schedule-rule.png[]
+. Select how often the rule runs.
+. Optionally, add look-back time to the rule. When defined, the rule searches 
+indices with the additional time.
++
+For example, if you set a rule to run every 5 minutes with an additional
+look-back time of 1 minute, the rule runs every 5 minutes but analyses the 
+documents added to indices during the last 6 minutes. This overlap helps ensure 
+signals are issued even when incoming data is delayed.
+. Save the rule with or without activation.
++
+NOTE: When you activate a rule it is queued, and its schedule is determined by 
```

Comment:
Add a comma after "rule,"

Reply:
I would rework this sentence to, "When you activate a rule, the rule places in a queue. The rule's initial run time determines its schedule. For example, if you activate a rule that runs every 5 minutes at 14:02, but it does not run until 14:04, the rule runs again at 14:09."